_overviews/collections-2.13/conversions-between-java-and-scala-collections.md

```diff
@@ -27,8 +27,8 @@ Sometimes you might need to pass from one collection framework to the other. For
 
 To enable these conversions, simply import them from the [JavaConverters](https://www.scala-lang.org/api/{{ site.scala-version }}/scala/collection/JavaConverters$.html) object:
```

Comment:
To make this PR complete this line and the second paragraph of the page should also be changed.

Reply:
Sorry if I'm missing something, but what updated URL should we be using in place of this?

Reply:
I guess the link should become `[CollectionConverters](https://scala-lang.org/api/{{ site.scala-version }}/scala/jdk/CollectionConverters$.html)`